os/net/lwip/src/api/sockets.c

```diff
@@ -1265,8 +1299,8 @@ int lwip_socket(int domain, int type, int protocol)
 		set_errno(ENOBUFS);
 		return -1;
 	}
-
 	i = alloc_socket(conn, 0);
+	nvdbg("socket = %d, pid = %d, %s: %s: %d\n", i, getpid(), __FILE__, __LINE__);
```

Comment:
Are there many `lwip_socket` functions? I don't know why you need __FILE__ and __LINE__. If there is only one, I think they are unnecessary.

Reply:
corrected.